file_formats/fip.md

```diff
@@ -14,77 +14,98 @@ Following SciComp standards, FIP data should be saved in their own folder named
 
 ```plaintext
 📦 fib
-┣ FIP_DataG_2024-06-05T08_25_33.csv
-┣ FIP_DataIso_2024-06-05T08_25_33.csv
-┣ FIP_DataR_2024-06-05T08_25_33.csv
-┣ FIP_RawG_2024-06-05T08_25_33.bin
-┣ FIP_RawIso_2024-06-05T08_25_33.bin
-┣ FIP_RawR_2024-06-05T08_25_33.bin
-┣ FIP_ROIsG-Iso_2024-06-05T08_25_33.csv
-┗ FIP_ROIsR_2024-06-05T08_25_33.csv
+┣ 📂 fip_<foo>
+┃ ┣ green.csv
+┃ ┣ red.csv
+┃ ┣ iso.csv
+┃ ┣ green.bin
+┃ ┣ red.bin
+┃ ┣ iso.bin
+┃ ┣ camera_green_iso_metadata.csv
+┃ ┣ camera_red_metadata.csv
+┃ ┗ regions.json
+┗ 📂 fip_<bar>
+  ┣ green.csv
+  ┣ <...>
+  ┗ regions.json
 ```
-The file names are `FIP_[channel-name]_[datetime].csv`.  The `[channel-name]` token can be one of the following:
 
-* `DataG`: data for green channel
-* `DataR`: data for red channel
-* `DataIso`: data for isosbestic channel
+Data is generally organized by the emission channel that gave rise to the data (`green`, `red`, and `iso`), respectively. For details on the rig setup, please refer to the [data acquisition repository](https://github.com/AllenNeuralDynamics/FIP_DAQ_Control).
+
+#### Fluorescence data
+
+Each fiber photometry session will primarily be analyzed by using the average signal from regions of interest (ROIs) placed on top of raw video frames during acquisition. To simplify analysis, we average the signal from all pixels within the ROIs during online acquisition and make it available as time series data in `CSV` files. These are `green.csv`, `red.csv`, and `iso.csv` files, respectively. All files share the same format, where each row corresponds to a single frame of the video and each column can be described as follows:
+
+* `ReferenceTime` Time of the trigger given by hardware (Harp)
+* `CameraFrameNumber` Frame counter given by the camera API
+* `CameraFrameTime` Frame acquisition time given by the camera API
+* `Background` CMOS dark count floor signal
+* `Fiber_0` Average signal values for Fiber_0's selected ROI
+* `<...>` (Variable number of columns)
+* `Fiber_N` Average signal values for Fiber_N's selected ROI.
+
+#### Raw sensor data
+
+Raw sensor data (i.e., camera frames) that generated the fluorescence data is saved in raw binary files. These files share the same naming convention as the fluorescence data files, but with a `.bin` extension. During acquisition, operators place circular ROIs over the images, and photometry readouts are obtained by averaging the signal inside these regions.
 
-#### Photometry Readout CSV files
+To open these files, users need additional information to parse the binary data. Data is stored in a `ColumnMajor` layout format, where each frame can be parsed with the information available in the corresponding `.meta` file. Each `.meta` file contains a single `JSON` object with the following fields:
```

Comment:
can this just be `metadata.json`, so file type is clear?

Reply:
green_meta.json?